pyaedt/modeler/Primitives3D.py

```diff
@@ -1050,8 +1050,10 @@ def create_udm(self, udmfullname, udm_params_list, udm_library="syslib"):
         vArg1.append("ConnectionID:=")
         vArg1.append("")
         oname = self.oeditor.CreateUserDefinedModel(vArg1)
+        obj_list = self.oeditor.GetPartsForUserDefinedModel(oname)
+        self.modeler.user_defined_models[oname] = obj_list
```

Comment:
@Samuelopez-ansys When possible, we should avoid to put list in the name of a variable, but I understand that sometimes it can be useful.
```suggestion
        model_parts = self.oeditor.GetPartsForUserDefinedModel(oname)
        self.modeler.user_defined_models[oname] = model_parts
```